README.md

```diff
@@ -149,34 +149,25 @@ The quadtree function splits the region of points into bins recursively based on
 
 ```
 qiime coordinates quadtree \
-    --m-metadata-file q2_coordinates/tests/data/chardonnay_sample_metadata.txt \
+    --m-metadata-file chardonnay_sample_metadata.txt \
     --p-y-coord latitude \
     --p-x-coord longitude \
     --p-threshold 50 \
     --output-dir test
 ```
 
 ### Visualizing quadtrees
-Quadtrees can easily be visualized using a mix of python and other existing qiime2 plugins. Some ideas inclued:
-1. using q2-coordinates draw map to disply the sample positions and color code them by respective quadrant (i.e: set the "column" parameter to the split-depth you want to look at)
-2. using q2-empress to view and navigate the tree to see number and size of splits.
-
-
-#### in python
-        import qiime2
-        import skbio
-        ar = qiime2.Artifact.load('test\tree.qza')
-        tree = ar.view(skbio.TreeNode)
-        for node in tree.traverse():
-            if node.length is None:
-                node.length = 1.0
-        new_ar = qiime2.Artifact.import_data('Phylogeny[Rooted]', tree)
-        new_ar.save('test\new_tree.qza')
-#### then using qiime2 cli
-            qiime empress tree-plot \
-            --i-tree test\new_tree.qza \
-            --m-feature-metadata-file q2_coordinates/tests/data/chardonnay_sample_metadata.txt \
-            --output-dir empress
+Quadtrees can easily be visualized using a mix of python and other existing qiime2 plugins. Some ideas include:
+1. using q2-coordinates draw map to display the sample positions and color code them by respective quadrant (i.e: set the "column" parameter to the split-depth you want to look at)
```

Comment:
"using q2-coordinates draw map" --> "using `q2-coordinates draw-map` or `draw-interactive-map`"

"by respective quadrant" --> "by their respective quadrants"

Reply:
instead of the parenthesis, it might just be simpler to give an example command here using the chardonnay dataset.